core/src/main/java/org/pentaho/di/core/vfs/KettleVFS.java

```diff
@@ -116,6 +116,14 @@ public static KettleVFS getInstance() {
     return kettleVFS;
   }
 
+  /** only use this method as a last resort if you don't yet have a variables object.  Since VFS connections can be used
+   * with variables for the dialog fields, it is important to pass the Variables to getFileObject.  Failure to do this
+   * means any settings for the job/ktr will not be available for variables substitution.
+   * Use getFileObject( String vfsFilename, VariableSpace space ), instead and pass the steps/entries variable space.
+   * @param vfsFilename
+   * @return
+   * @throws KettleFileException
+   */
   public static FileObject getFileObject( String vfsFilename ) throws KettleFileException {
```

Comment:
Should we deprecate this method, maybe?  Or are there still too many legitimate uses for it?